deployment/evm/verify.ts

```diff
@@ -13,15 +16,26 @@ evm.runOnEvms("bytecode-verification-token-router", async (chain, signer, log) =
     return;
   }
 
+  // This is to just have default values when json does not contain the required data
+  const deployer = "0x085AEF9C8a26cF4b95E7eF1Ebb27E62615fc881D";
```

Comment:
maybe we want to use the 0 address here instead? might be confusing to just see address that's not correct

Reply:
makes sense, I'll push the change